README.md

```diff
@@ -67,6 +67,21 @@ In any Swift file where you want to use BonMot, simply `import BonMot`. In Objec
 
 The basic object in BonMot is `BONChain`, which allows you quickly construct attributed strings. You can create a chain with a normal `[[BONChain alloc] init]`, but it's easier to just use `[BONChain new]` or the even shorter and technically valid `BONChain.new`:
```

Comment:
#nth change this to Swift: `BONChain()`, and bump the Obj-C to inside that section. You can probably just delete it, since the Obj-C code snippet already has example usage.